lib/staccato/V4/tracker.rb

```diff
@@ -12,11 +12,19 @@ class Tracker
     # @param measurement_id [String] the measurement id from GA
     # @param api_secret [String] the required api secret key
     # @param client_id [String, nil] unique value to track user sessions
+    # @param user_id [String, nil] unique value to track user
     # @param options [Hash]
-    def initialize(measurement_id, api_secret, client_id = nil, options = {})
+    def initialize(
+      measurement_id,
+      api_secret,
+      client_id = nil,
+      user_id = nil, 
+      options = {}
+    )
```

Comment:
Question for @base10 - Is this argument format going to be compatible with Ruby 3?  If not, do we care?